include/ignition/gazebo/rendering/RenderUtil.hh

```diff
@@ -138,7 +138,7 @@ inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
 
     /// \brief Get the entities currently selected, in order of selection.
     /// \return Vector of currently selected entities
-    public: std::vector<Entity> SelectedEntities() const;
+    public: const std::vector<Entity> &SelectedEntities() const;
```

Comment:
@chapulina Does ignition style suggest that the `&` be next to the type or the method here, do you know? I just switched it to this after seeing some other getters that return references above.

Reply:
It goes next to the variable, see the [style guide](https://ignitionrobotics.org/docs/all/contributing#style-guides). This is something carried over from Gazebo classic and we have no linters to catch yet.